src/ethPriceLoop/eth_usd_price.ts

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+import { getEthPriceCount, persistEthPrices } from './db.eth_price';
+import { Logger } from '../logging';
+import { EthPrice } from '../entities/IEthPrice';
+import { Time } from '../time';
+
+const MOBULA_HISTORIC_URL =
+  'https://api.mobula.io/api/1/market/history?asset=Ethereum&from=1633046400000';
+
+const MOBULA_CURRENT_URL =
+  'https://api.mobula.io/api/1/market/data?asset=Ethereum';
+
+interface HistoricResponse {
+  data: {
+    price_history: [number, number][];
+  };
+}
+
+interface CurrentResponse {
+  data: {
+    price: number;
+  };
+}
+
+const logger = Logger.get('ETH_PRICE');
+
+export async function getNewEthUsdPrice(reset: boolean) {
```

Comment:
Using get and fetch prefixes in modifying functions. Still not a fan of that :)

```
getNewUsdPrice -> synchroniseEthUsdPrice
fetchHistoricData -> synchroniseHistoricEthUsdPriceData
fetchCurrentData -> synchroniseLatestEthUsdPriceData
```

Reply:
updated - used 'sync' to have a bit shorter name